pkg/dns/aws/dns.go

```diff
@@ -190,7 +190,6 @@ func (m *Provider) getZoneID(zoneConfig configv1.DNSZone) (string, error) {
 func (m *Provider) getLBHostedZone(name string) (string, error) {
 	m.lock.Lock()
 	defer m.lock.Unlock()
-
```

Comment:
Please don't delete this line.  See the comments on https://github.com/openshift/cluster-ingress-operator/commit/a3b29e2e4598581dea2a1d2a9789828f2bf76a8f.

Reply:
done